### DISCUSSION TITLE
Red Hat Linux 7.8安装ApiSix报错

### DISCUSSION THREAD

**Discussion by beidouyunyun** [apache/apisix#7135] [Q&A]
系统版本： Red Hat Enterprise Linux Server release 7.8 (Maipo)
安装方式：离线安装 yum install *.rpm
报错内容：
![image](https://user-images.githubusercontent.com/18464511/170404358-840c03de-35cc-4291-ac2f-a5fbfa914e8a.png)

**Comment by tzssangglass** [comment]
see: https://github.com/apache/apisix/blob/master/docs/en/latest/installation-guide.md#installation-via-rpm-offline-package

Or you need to provide more reproduction steps.
